src/lib/network_pool/snark_pool.ml

```diff
@@ -330,15 +330,21 @@ module Make (Transition_frontier : Transition_frontier_intf) :
                       work ) ] ;
           `Statement_not_referenced )
 
+      (* At this point, verification already has happened *)
```

Comment:
You mean after the function is called?

Reply:
I was confused about what the function was doing when I wrote the comment :)